aptos-move/move-examples/fungible_asset/sources/premined_managed_coin.move

```diff
@@ -0,0 +1,63 @@
+/// This module shows an example how to issue preminted coin with only `transfer` and `burn` managing capabilities.
+/// It leveraged `managed_fungible_asset` module with only `TransferRef` and `BurnRef` stored after pre-minting a
+/// pre-defined totally supply to a reserve account. After the initialization, the total supply can increase by no means
+/// since `MintRef` of this fungible asset does not exist anymore.
+/// The `init_module()` code can be modified to customize the managing refs as needed.
+module example_addr::premined_managed_coin {
```

Comment:
Weird that this a typo. Can you rebase this with the other PR?

Reply:
I caught that in another PR but not this stacked one. ;(